tests/page-objects/collection-browser.ts

```diff
@@ -71,12 +71,11 @@ export class CollectionBrowser {
     ).toContain(checkFilterText);
   }
 
-  async validateURLParamsWithSortFilter(filter: SortFilter, order: SortOrder) {
+  async urlParamsWithSortFilter(filter: SortFilter, order: SortOrder) {
     const sortFilterURL =
       order === 'descending'
         ? `-${SortFilterURL[filter]}`
         : SortFilterURL[filter];
-    const urlPatternCheck = new RegExp(`sort=${sortFilterURL}`);
-    await expect(this.page).toHaveURL(urlPatternCheck);
+    return new RegExp(`sort=${sortFilterURL}`);
   }
```

Comment:
Suggestion: I think we could name this method more clearly. Since its purpose is now to produce a URL-testing regexp, maybe `getURLSortValidationPattern()`?

Also, it no longer needs to be `async`, and could perhaps be a static method too.